src/main/java/com/snowflake/kafka/connector/SnowflakeSinkTask.java

```diff
@@ -388,29 +388,33 @@ private static void waitFor(Supplier<Boolean> func)
     throw new TimeoutException();
   }
 
-  private static long getExecutionTimeSec(long startTime, long endTime) {
-    return (endTime - startTime) / 1000;
+  private static long getDurationFromStartMs(long startTime) {
+    final long currTime = System.currentTimeMillis();
+    return currTime - startTime;
   }
 
   void logWarningForPutAndPrecommit(long startTime, int size, String apiName) {
-    long executionTime = getExecutionTimeSec(startTime, System.currentTimeMillis());
-    if (executionTime > 300) {
+    long executionTimeMs = getDurationFromStartMs(startTime);
+    long executionTimeSec = executionTimeMs / 1000;
+    if (executionTimeSec > 300) {
       // This won't be frequently printed. It is vary rare to have execution greater than 300
       // seconds.
       // But having this warning helps customer to debug their Kafka Connect config.
       this.DYNAMIC_LOGGER.warn(
-          "{} {}. Time: {} seconds > 300 seconds. If there is CommitFailedException in the log or"
-              + " there is duplicated records, refer to this link for solution: "
+          "{} {}. Time: {} milliseconds = {} seconds > 300 seconds. If there is"
+              + " CommitFailedException in the log or there is duplicated records, refer to this"
+              + " link for solution: "
               + "https://docs.snowflake.com/en/user-guide/kafka-connector-ts.html#resolving-specific-issues",
           apiName,
           size,
-          executionTime);
+          executionTimeMs,
+          executionTimeSec);
     } else {
-      this.DYNAMIC_LOGGER.debug(
-          "successfully called {} with {} records, execution time: {} seconds",
+      this.DYNAMIC_LOGGER.info(
+          "successfully called {} with {} records, execution time: {} milliseconds",
```

Comment:
Maybe try to reduce the words here if the concern is length

Reply:
changed to ms